docs/maya/glue.md

```diff
@@ -0,0 +1,145 @@
+| **Triangulate Mesh**      | Boolean  | False | ✗ | Use the internally triangulated mesh to build constraints. |
+| **Glue Multiplier**       | Float    | 1.0   | ✓ | Sets the multiplier factor for the weights of the glue constraint. Has a range of [0.0, 1.0]. |
+| **Max Glue Distance**     | Float    | 1.0   | ✓ | Sets maximum distance at which a vertex has to be from neighbor surfaces to create a glue constraint. Depending on the scale of your creature, higher values might be required to guarantee dense glue connections to be created. Use the debugger to help you define the value that fits your creature the best. |
+| **Compression Multiplier** | Float   | 1.0   | ✓ | Sets the scaling factor applied to the compression resistance of every point. Has a range of \[0.0, 2.0\]. The upper limit is soft, higher values can be used. |
+| **Stretching Multiplier** | Float    | 1.0   | ✓ | Sets the scaling factor applied to the stretching resistance of every point. Has a range of \[0.0, 2.0\]. The upper limit is soft, higher values can be used. |
+
+### Node Attributes
+| Name | Type | Default | Animatable | Description |
+| :--- | :--- | :------ | :--------- | :---------- |
+| **Envelope** | Float | 1.0 | ✓ | Specifies the deformation scale factor. Has a range of \[0.0, 1.0\]. The upper and lower limits are soft, values can be set in a range of \[-2.0, 2.0\]|
+
+### Debug Attributes
+| Name | Type | Default | Animatable | Description |
+| :--- | :--- | :------ | :--------- | :---------- |
+| **Debug**       | Boolean      | False                | ✓ | Enable or Disable the debug functionalities in the viewport for the AdnGlue node. |
+| **Feature**     | Enumerator   | Distance Constraints | ✓ | A list of debuggable features for this node.<ul><li>Distance Constraints: Draw *Distance Constraint* connections representing the constrained pair of vertices in the simulated mesh.</li><li>Shape Preservation: Draw *Shape Preservation* connections between the vertices adjacent to the vertices with this constraint.</li><li>Glue Constraints: Draw *Glue* connections between every vertex and its closest point on the closest neighbor surface.</li></ul> |
+| **Width Scale** | Float        | 3.0                  | ✓ | Modifies the width of all lines. |
+| **Color**       | Color Picker | Red                  | ✓ | Selects the line color from a color wheel. Its saturation can be modified using the slider. |
+
+## Attribute Editor Template
+
+<figure markdown>
+  ![glue editor](images/glue_attribute_editor_00.png) 
+  <figcaption><b>Figure 1</b>: AdnGlue Attribute Editor.</figcaption>
+</figure>
+
+## Paintable Weights
+
+In order to provide more artistic control, some key parameters of the AdnGlue solver are exposed as paintable attributes in the node. The Maya paint tool must be used to paint those parameters to ensure that the values satisfy the solver requirements.
+
+| Name | Default | Description |
+| :--- | :------ | :---------- |
```

Comment:
The entries of this table have to be aligned.

Reply:
All tables reviewed